src/plugins/console/public/lib/kb/kb.js

```diff
@@ -129,23 +90,27 @@ function loadApisFromJson(
   urlParametrizedComponentFactories,
   bodyParametrizedComponentFactories
 ) {
-  urlParametrizedComponentFactories =
-    urlParametrizedComponentFactories || parametrizedComponentFactories;
-  bodyParametrizedComponentFactories =
-    bodyParametrizedComponentFactories || urlParametrizedComponentFactories;
-  const api = new Api(urlParametrizedComponentFactories, bodyParametrizedComponentFactories);
-  const names = [];
-  _.each(json, function (apiJson, name) {
-    names.unshift(name);
-    _.each(apiJson.globals || {}, function (globalJson, globalName) {
-      api.addGlobalAutocompleteRules(globalName, globalJson);
+  try {
```

Comment:
We need a try-catch clause here, otherwise if there is an error while loading definitions, the process fails silently.